typescript/elasticbeanstalk/elasticbeanstalk-bg-pipeline/index.ts

```diff
@@ -0,0 +1,64 @@
+import cdk = require('@aws-cdk/cdk');
+import cp = require('@aws-cdk/aws-codepipeline');
+import cc = require('@aws-cdk/aws-codecommit');
+import lambda = require('@aws-cdk/aws-lambda');
+import s3 = require('@aws-cdk/aws-s3');
+
+export class CdkStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+		super(scope, id, props);
+
+		//objects for access parameters
+		const construct = new cdk.Construct(this, 'construct');
```

Comment:
You should not create objects like this. For `construct` you can substitute `this`, and for `node` you can substitute `this.node`.

Reply:
Leveraging `this.node` to access parameters in both samples now.